src/App.jsx

```diff
@@ -14,45 +16,111 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
+const SORT__ALPHABETICALLY = 'alphabet';
+const SORT__BY__LENGTH = 'length';
+const getPreparedGoods = (goods, visibleGoods, reversed) => {
+  const copyGoods = [...goods];
+
+  if (visibleGoods) {
+    copyGoods.sort((good1, good2) => {
+      switch (visibleGoods) {
+        case SORT__ALPHABETICALLY: return good1.localeCompare(good2);
+        case SORT__BY__LENGTH: return good1.length - good2.length;
+
+        default: return 0;
+      }
+    });
+  }
+
+  if (reversed) {
+    switch (reversed) {
+      case 'reverse':
+        return copyGoods.reverse();
+
+      default: return copyGoods;
+    }
+  }
+
+  return copyGoods;
```

Comment:
```suggestion
  return isReversed ? copyGoods.reverse() : copyGoods;
```
What's the purpose of switch here? It can be simplified